src/App.js

```diff
@@ -38,6 +38,22 @@ const updateExchangeRates = () => {
         .write()
     })
   })
+
+  axios.get('https://api.cryptonator.com/api/ticker/eth-eur').then(response => {
```

Comment:
Can you change the ticker to `eur-eth`? That way you won't have to divide 1 bu the ticker value 😊